lexicon/providers/safedns.py

```diff
@@ -8,14 +8,14 @@
 
 LOGGER = logging.getLogger(__name__)
 
-NAMESERVER_DOMAINS = ['graphiterack.com']
+NAMESERVER_DOMAINS = ['ukfast.net']
```

Comment:
Are we sure that the previous nameserver domains is not use anymore? Note that we can have several values in the array, the `auto` provider (that consumes the `NAMESERVER_DOMAINS` values in other providers), is able to handle this situation, and will check against each domain provided here.

Reply:
The `graphiterack.com` domain is used for our internal recursive nameservers, not the authoritative ones, so I thought it best to leave them off. Any the nameserver entries for customers that we cost domains for use `ns0.ukfast.co.uk` and `ns1.ukfast.co.uk`